src/project-tests/calculator-tests.js

```diff
@@ -137,13 +138,14 @@ export default function createCalculatorTests() {
       containing a "." (decimal point) symbol with a corresponding
       id="decimal"`,
       function() {
+        var element = document.getElementById(_dec);
         assert.isNotNull(
-          document.getElementById(_dec),
+          element,
           'id="decimal" is not yet defined '
         );
 
         assert.strictEqual(
-          document.getElementById(_dec).innerHTML,
+          element.localName === 'input' ? element.value : element.innerText,
```

Comment:
Is there are reason to change from using `innerHTML` to `innerText`? Everywhere else in the code (example https://github.com/freeCodeCamp/testable-projects-fcc/blob/78310151942f56a99713d01abc9079ee8ffc3442/src/project-tests/calculator-tests.js#L181) we are using `innerHTML`.

I'm not opposed to this change as long as there is a good reason. More info about the differences here:

https://stackoverflow.com/questions/19030742/difference-between-innertext-and-innerhtml#19032002

https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent

Let's discuss which you think is the best to use in this code and why. If we do decide to go with `innerText`, we should be consistent and make this change everywhere.

Reply:
I think the best to use "innerText" in this code because we only want to check that a user sees some symbol on the screen, and not how it is coded.

This does not correspond to the issue, so I decided to propose it for the next PR.

Reply:
@ValeraS I like `innerText` more too for this test suite. Mostly because `innerText` will not return text for invisible items, and the items should be visible for the tests to pass. But for this PR I would like to keep the code consistent. I would suggest either using `innerHTML` above for now and then submitting a separate PR to change all occurrences of of `innerHTML` to `innerText`. Or just changing everything in this PR. The cleanest approach would be two separate PRs. But it's a small change so doing everything in one PR is ok. Up to you.